src/components/Kv/KvCarousel.vue

```diff
@@ -54,5 +54,6 @@ export default {
 	position: relative;
 	overflow: hidden;
 	width: 100%;
+	height: 100%;
```

Comment:
Is this going to have any unintended side effects? I didn't include this originally so that the content of the slides could drive the height of the carousel as opposed to needing to set the height in a wrapper.

Reply:
Seems OK so far. Currently KvCarousel is only used in KvHero and KivaContentBlock and those two look OK. Height is always kindof confusing to me in CSS but I think the content of the slides will still be driving that 100%.

Reply:
☝️ @ryan-ludwig 
FYI: I had to add a slew of media queries in KivaContentBlock.vue in order to get the height right across all the screen sizes we support. That fix was definitely less than ideal.

Reply:
Ok, this is fine for now then